docs-site/website/versioned_docs/version-2.0.0/onnx.md

```diff
@@ -1,6 +1,7 @@
 ---
-id: onnx
+id: version-2.0.0-onnx
```

Comment:
@joddiy hi, if i understand prof wang's suggestion, we can ignore the update for `versioned_docs` for v2.0.0, as this update only applies to v2.0+

Reply:
yes. version 2.0.0 is already fixed. all new files or updates will be in v2.1